scipy/stats/morestats.py

```diff
@@ -1474,6 +1478,10 @@ def ansari(x, y):
            methods.  3rd ed. Chapman and Hall/CRC. 2001.  Section 5.8.2.
 
     """
+    if alternative not in ('two-sided', 'less', 'greater'):
+        raise ValueError("alternative not recognized\n"
```

Comment:
no need for ``\n`` here, you can replace it with ``, `` to keep the output on a single line.